### DISCUSSION TITLE
Report filter: No tags option

### DISCUSSION THREAD

**Discussion by BoxOfSnoo** [solidtime-io/solidtime#237] [Feature requests]
I would find it very useful to view entries without tags - or for that matter, no project specified. This would help find and organize entries that are incomplete.  I also use tags when invoicing, so this shows me open time records.

**Comment by schnere** [comment]
I want to add here: Filter option for `tag is not set`
Usecase: Tag entries as `invoiced` and then filter by tag `invoiced` not set for next billing period.

**Comment by Onatcer** [comment]
This is now released in [v0.11](https://github.com/solidtime-io/solidtime/releases/tag/v0.11.1) Supporting projects, tasks, clients and tags.

**Reply by BoxOfSnoo** [reply]
I just had to come back to say how extremely happy I am with this feature. It really helps me dial down to the tasks I really need to focus on! Thank you!
